lib/nostrum/consumer.ex

```diff
@@ -99,19 +104,27 @@ defmodule Nostrum.Consumer do
   @type guild_member_remove ::
           {:GUILD_MEMBER_REMOVE,
            {guild_id :: integer, old_member :: Nostrum.Struct.Guild.Member.t()}, WSState.t()}
+  @typedoc """
+  Dispatched when a guild member is updated.
+  `old_member` will be `nil` when the pre-update member could not be fetched from the cache.
```

Comment:
Could you add a new line between the two sentences? ExDoc uses the first line to generate summaries, so we should try to keep it away from the rest of the provided info just for clarity.